utilities/ovs-p4ctl.in

```diff
@@ -47,20 +47,20 @@ ovs = None
 USAGE = "ovs-p4ctl: P4Runtime switch management utility\n" \
         "usage: ovs-p4ctl [OPTIONS] COMMAND [ARG...]\n" \
         "\nFor P4Runtime switches:\n" \
-        "  show SWITCH                         show P4Runtime switch " \
+        "grpc-addr  show SWITCH                         show P4Runtime switch " \
```

Comment:
Couple of comments:
1. Better to introduce this parameter as key value pair. Ex: grpc-serv-ip=xxx.xxx.xxx.xxx.
2. Make this as an optional argument. 
3. If this parameter is not provided, continue with default address that is localhost.

Reply:
Done